### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by ctf-umbraco**

### CODE REVIEW

File: Umbraco-Uno/Creating-Content/index.md [hunk 1]
```diff
@@ -58,3 +58,7 @@ The Rich Text Editor or RTE as people call it is a great way to create content.
 When you're creating content in Umbraco Uno you will primarily be working with Widgets. These allow for a work process that is very straight forward, and they can be arranged in a grid for more complexity.
 
 For a full overview of the Widgets provided with Umbraco Uno see the [Widgets](../Widgets) section in the Uno-pedia.
+
+## [Working with Forms](../Working-with-Forms)
+
+If you are new to Umbraco Forms and want to learn more about how to work with them then you can read more about them in the Working with Forms section.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Creating-Content/index.md:64]:
```suggestion
If you are new to Umbraco Forms, and want to learn more about how to work with them, you can read more about the workflow in the Working with Forms section.
```

File: Umbraco-Uno/Widgets/Umbraco-Form/index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ versionFrom: 8.0.0
 
 The Umbraco Form widget, gives you the ability to add a custom form to your site. It is important to note that a custom form needs to have been made using Umbraco Forms before this widget can be used.
 
-You can create a form by selecting the Forms section in the top menu in your backoffice. Learn more about how to create forms in the [Umbraco Forms](../../../Add-ons/UmbracoForms/Editor/Creating-a-Form) section.
+You can create a form by selecting the Forms section in the top menu in your backoffice. Learn more about how to create forms in the [Working with Forms](../../Working-with-Forms) section.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Widgets/Umbraco-Form/index.md:9]:
```suggestion
You can create a form by navigating to the Forms section in the top menu of your backoffice. Learn more about how to create forms in the [Working with Forms](../../Working-with-Forms) section.
```

File: Umbraco-Uno/Working-with-Forms/Replies/index.md [hunk 3]
```diff
@@ -11,11 +11,11 @@ What you will want to do is:
 * Then in the big window located in the middle of the screen, a box will show you how many replies you have.
 * Clicking this box will bring you to an overview of all the replies, and you can click the name of the reply to get into each of them to see them filled out.
 
-![One form](images/One-form.png)
+![One form](images/Replies-amount.png)
 
 If you have more then one form, there will be a box for each of the forms. If you have more then four forms, you will have the option to ***show all*** as only four will be shown as default.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Working-with-Forms/Replies/index.md:16]:
```suggestion
If you have more than one form, there will be a box for each of the forms. If you have more than four forms, you will have the option to ***show all*** as only four will be shown as default. 
```

File: Umbraco-Uno/Working-with-Forms/Umbraco-Forms-widget/index.md [hunk 4]
```diff
@@ -6,12 +6,15 @@ versionFrom: 8.0.0
...
+
 The first option is to add the widget called Umbraco Forms. In this widget, you will get the option to select which form you want to add. You can read about this widget's settings in the article about the [Umbraco Forms widget](../../Widgets/Umbraco-Form).
 
-![Regular Umbraco Forms](images/Regular-form.png)
+![Regular Umbraco Forms](images/Umbraco-forms.png)
 
 The second option is to add the form through a Grid widget. The setup here is slightly different than with the Umbraco Forms widget. However, the way you add the form is the same as the first option. Go to the section of the widget that says ***Form*** and then click "Add" to select the form you want.
 
-![Grid Umbraco Forms](images/Grid-forms.png)
+![Grid Umbraco Forms](images/Grid-form-filled.png)
```
Comment (Jay-umbr) [inline Umbraco-Uno/Working-with-Forms/Umbraco-Forms-widget/index.md:19]: You say "in the next article", but is it going to be on the same page? If the link just before that points to the article you mentioned  I would just remove "from your form which is the next article" part

File: Umbraco-Uno/Working-with-Forms/workflow/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+versionFrom: 8.0.0
+---
+
+# What is a workflow
+
+A workflow, in this case, is the flow that happens from you press the submit form button on the frontend until it ends in your email (or wherever you assigned it to arrive)
```
Comment (Jay-umbr) [inline Umbraco-Uno/Working-with-Forms/workflow/index.md:7]:
```suggestion
A workflow, in this case, is the flow that starts when you press the "submit form" button on the frontend, and ends with the form reply being sent to your email (or wherever you assigned it to arrive)
```

File: Umbraco-Uno/Working-with-Forms/workflow/index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+The most common Workflow is to have the user press the submit button, then a thank you message pops up, and then the email is sent to a designated mail inbox. However, you could choose to have it sent to, for example, slack if you wanted. In this guide, you will learn to set-up a basic workflow for your forms.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Working-with-Forms/workflow/index.md]:
```suggestion
The most common Workflow is to have the user press the submit button, present a thank you message, and then send the email to a designated mail inbox. However, you could choose to have it sent to, for example, a Slack channel if you wanted. In this guide, you will learn to set-up a basic workflow for your forms.
```

File: Umbraco-Uno/Working-with-Forms/workflow/index.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+:::tip
+This requires you to have set up a form if you dont have a form set-up yet you can learn how to do that in the [Setting up a form](../Setting-up-a-form/index.md) setion.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Working-with-Forms/workflow/index.md]:
```suggestion
This requires you to have set up a form. If you don't have a form set-up yet, you can learn how to do that in the [Setting up a form](../Setting-up-a-form/index.md) section.
```

File: Umbraco-Uno/Working-with-Forms/workflow/index.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+In this part, we will learn how to enter the configuration menu for workflows.
+We will be working with a form that contains the following:
+
+* A Name field using a short answer
+* An Email field using a short answer
+* A Question using a long answer
+
+This is how you get into the configuration of your workflow:
+
+1. In the Forms section of the backoffice.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Working-with-Forms/workflow/index.md:26]:
```suggestion
1. Navigate to the Forms section of the backoffice.
```

File: Umbraco-Uno/Working-with-Forms/workflow/index.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+2. Here, select the form you want to add a workflow to.
+3. In the bottom of the page, you will find a section containing a checkmark saying ***on submit***
+4. Click ***Configure Workflow***
+5. The pop op window will contain the Workflow that currently exists
+6. Select the one that says *Sending template email to*
+
+### Filling in the Workflow
+
+In this section, we will make the Workflow send to your email of choice. Please note that this is for a workflow that contains a template. To set up the one without a template, it is the same except the 8th step.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Working-with-Forms/workflow/index.md]:
```suggestion
In this section, we will make the Workflow send an email to your inbox of choice. Please note that this is for a workflow that contains a template. To set up the one without a template, it is the same approach except for the 8th step.
```

**Review by Jay-umbr — CHANGES_REQUESTED** [review]
Lengthy article, so I had suggested some changes - mostly how I would phrase some things, and fixed 2-3 grammar mistakes too. Quite a nice thing, overall!

File: Umbraco-Uno/Working-with-Forms/Workflow/index.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+5. The pop op window will contain the workflow that currently exists
+6. Select the one that says *Sending template email to*
+
+### Editing the existing workflow
+
+If you choose an ***empty form*** when setting up a new form, it will come with a Working workflow. However, you will have to make a slight change to it before it functions optimally.
+This can be done in the configuration menu for workflows.
+
+1. Click the box ***Send template email to***
+2. In the field called ***Sender email***, you should add something like Noreply@something.something.
```
Comment (sofietoft) [inline Umbraco-Uno/Working-with-Forms/Workflow/index.md:41]:
```suggestion
2. In the field called ***Sender email***, you should add something like Noreply@something.something
```

File: Umbraco-Uno/Working-with-Forms/Workflow/index.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+3. It is recommended to choose ***Igloo template***
+4. Hit submit and then hit submit in the next menu as well
```
Comment (sofietoft) [inline Umbraco-Uno/Working-with-Forms/Workflow/index.md:43]: Would be good with a quick ending to this.

File: Umbraco-Uno/Working-with-Forms/Workflow/index.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+
+### Creating a custom workflow
+
+In this section, you will learn how to make a custom workflow that sends an email to your inbox of choice. 
+
+1. First, give the workflow the name you want it to have in the ***Workflow Name*** field.
+2. Put a checkmark in the ***Active**** so that the workflow is active.
+3. The ***Include Sensitive Data*** will be left unchecked.
+4. Ensure that the ***Email*** is set to the email which should receive replies from forms.
+5. Next up is the ***Sender Email*** here. You can add something like Noreply@something.com.
```
Comment (sofietoft) [inline Umbraco-Uno/Working-with-Forms/Workflow/index.md:53]:
```suggestion
5. Next up is the ***Sender Email***. Here you can add something like Noreply@something.com.
```

File: Umbraco-Uno/Working-with-Forms/Workflow/index.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+6. If you want to reply directly to the customer from the forms reply, you can add {email} to the ***ReplyToEmail*** this will fetch the email from the field called email in the form.
+7. Next, you have the option to choose the subject of the email.
+8. In the next field, you can pick a template. It is recommended to use ***Igloo-template.cshtml***.
+9. Finally, leave ***Attachments*** unchecked unless you need the users to send attachments to your inbox.
```
Comment (sofietoft) [inline Umbraco-Uno/Working-with-Forms/Workflow/index.md:57]:
```suggestion
9. Finally, leave ***Attachments*** unchecked, unless you have an upload-field in the form, and want to receive the uploaded files through the workflow.
```

File: Umbraco-Uno/Working-with-Forms/Workflow/index.md [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+3. It is recommended to choose ***Igloo template***
+4. Hit submit and then hit submit in the next menu as well
+
+### Creating a custom workflow
+
+In this section, you will learn how to make a custom workflow that sends an email to your inbox of choice. 
+
+1. First, give the workflow the name you want it to have in the ***Workflow Name*** field.
+2. Put a checkmark in the ***Active**** so that the workflow is active.
+3. The ***Include Sensitive Data*** will be left unchecked.
```
Comment (sofietoft) [inline Umbraco-Uno/Working-with-Forms/Workflow/index.md:51]:
```suggestion
3. The ***Include Sensitive Data*** will be left unchecked unless your form contains sensitive data.
```

File: Umbraco-Uno/Working-with-Forms/index.md [hunk 15]
```diff
@@ -27,3 +27,7 @@ In this section, you will learn to check the replies to the form you have added.
 ## [Recaptcha](Recaptcha)
 
 In this section, you can learn how to add Recaptcha to your forms.
+
+## [Workflows](Workflows)
+
+Here you can learn how to work with the excisting workflow or create your own.
```
Comment (sofietoft) [inline Umbraco-Uno/Working-with-Forms/index.md:33]:
```suggestion
Here you can learn how to work with workflows, and decide what should happen once visitors on your site fill in a form.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
I've made some suggestions.
Please take a look when you have time.

**Pull request merged**